src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -63,6 +65,49 @@ Amplify.configure({
 });
 ```
 
+### Lex V2 bot
+You can create Amazon Lex V2 chatbox in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex V2 Developer Guide](https://docs.aws.amazon.com/lexv2/latest/dg/getting-started.html).
```

Comment:
`chatbot` not `chatbox`

Reply:
I would mention here where to get the `botId` `aliasId` and `localeId` from the console

Reply:
Updated

Reply:
> I would mention here where to get the `botId` `aliasId` and `localeId` from the console

@abdallahshaban557

Reply:
I think I can fill in that information at a later time. I need to reformat this whole category either ways.